Coldwave/ColdwaveApp.swift

```diff
@@ -13,6 +17,11 @@ struct ColdwaveApp: App {
             CommandGroup(before: CommandGroupPlacement.newItem) {
                 Button(action: { openFolder() }, label: { Label("Open directory...", systemImage: "doc") })
                     .keyboardShortcut("o")
+
```

Comment:
This is to add a "modal" dialog to ask for a remote location (audio stream or remote file)